draft/examples/reconciliation-query-batch/invalid/misnamed-property.json

```diff
@@ -1,10 +1,12 @@
 {
   "queries": [
     {
-      "query": "Christel Hanewinckel",
       "type": "DifferentiatedPerson",
       "limit": 5,
       "props": [
+        {
+          "v": "Christel Hanewinckel"
```

Comment:
Why no property? Shouldn't it be `rdfs:label` or something similar?

Reply:
Let's keep the discussion in one place, below